cpg_infra/billing_aggregator/driver.py

```diff
@@ -169,31 +170,46 @@ def setup_gcp_cost_control(self):
         )
 
         # Deploy Cloud Function
+        env = {'SLACK_CHANNEL': slack_channel}
+        memory = 256
```

Comment:
Can we add the 'MB' to the end of this? (I'd be surprised if it didn't require it)

Reply:
Fixed.